### PR TITLE
added the first function.

### PR DESCRIPTION
**Pull request by Tommyl404**
can't check further with inputs, need to have function 3 online.
added a flag to the main menu so when case 11 is active it will be false and really terminate the script as it should.

### CODE REVIEW

**Comment by ShirGivon** [comment]
הי, עשיתי את פעולת ההדפסה, הוספתי גם את הפונקציות שאני צריך לעשות על הדרך. 3,4,9

**Comment by dudugvili** [comment]
@ShirGivon Please remove your changes from this Pull Request and make a new branch of your own with the desired changes

File: Ex1/main.java [hunk 1]
```diff
@@ -22,15 +22,15 @@ static void printMenu() {
...
+        boolean flag = true;
+        while (flag) {
         printMenu();
         String choice = scn.nextLine();
             switch (choice) {
                 case "1":
                     addContact(phone_book);
                     break;
-                case "2":
+               case "2":
```
Comment (dudugvili) [inline Ex1/main.java:36]: Unnecessary removal of space

File: Ex1/main.java [hunk 2]
```diff
@@ -59,8 +59,48 @@ public static void main(String[] args) {
...
+                    flag = false;
                     break;
             }
         }
     }
+
+    public static void addContact(ArrayList<Contact> phone_book)
+    {
+        Scanner sForAdd = new Scanner (System.in); //open a new scanner for input
+        /*a do-while loop for taking care of invalid phone numbers, using .next() for only
```
Comment (dudugvili) [inline Ex1/main.java:74]: For proper documentation please refer to this guidelines - 
https://www.oracle.com/il-en/technical-resources/articles/java/javadoc-tool.html

**Review by dudugvili — CHANGES_REQUESTED** [review]
Please see the comments I've wrote,
also - keep this naming convention:
SomeClass
someFunction
some_variable

**Comment by dudugvili** [comment]
@Tommyl404 
other than some documentation changes the code looks good.
Fix that and after Shir will remove his changes from here we will merge this PR.

**Comment by ShirGivon** [comment]
Created new branch function_349. Still need updates, documentation, and Naming conventions.

File: Ex1/Contact.java [hunk 3]
```diff
@@ -21,4 +21,5 @@ public void setName(String name) {
     public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
+
```
Comment (dudugvili) [inline Ex1/Contact.java:24]: Unnecessary

**Pull request closed without merging**